app/views/app_configs/_form.erb

```diff
@@ -48,6 +48,7 @@
 
   <fieldset>
     <legend>Reservation Settings</legend><br />
+    <%= f.input :requests_affect_availability, label: 'Allow requests to block equipment availability?', hint: 'When enabled, reservation requests will block equipment item availability before the request is approved.' %>
```

Comment:
I would maybe switch the word "block" with "reduce", or something along those lines. Block makes it sound like there won't be any availability, which is not the case. We can also simply use "affect"; what do you think?